app/models/sign_up_sheet.rb

```diff
@@ -103,24 +99,24 @@ def self.add_signup_topic(assignment_id)
     @duedates = {}
     return @duedates if @topics.nil?
     @topics.each_with_index do |topic, i|
-      @duedates[i] = duedate = {}
-      duedate['id'] = topic.id
-      duedate['topic_identifier'] = topic.topic_identifier
-      duedate['topic_name'] = topic.topic_name
+      @duedates[i] = due_date = {}
```

Comment:
`@duedates` can also be renamed as `@due_dates`.

Reply:
Sorry about the delay, just saw this.

Agreed. There were a couple of those, but I thought our time was better spent worrying about overt Code climate issues. The issue for me was that I'm pretty pedantic about English and what is cringe-worthy to me may not be to someone else. And I didn't want to fall down that rabbit hole.

We can certainly look at changing that after homework 4 is finished.